recipes/setuptools-github/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "setuptools-github" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf5ebf0ad81a872a9bb50123ff33387f82d5c80467949115cf06eca003479670
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+# test:
+#   requires:
+#     - pip
+#     - pytest
```

Comment:
We generally need some kind of test to validate that the created conda package isn't broken. Is there some kind of `import` we can do here or a command to run?

Reply:
done